string object_name
string object_type
---
bool success
